### DISCUSSION TITLE
Conbee II/Deconz integration, not showing LQI for many devices

### DISCUSSION THREAD

**Discussion by dekesone** [Koenkk/zigbee2mqtt#8484] [Devices]
Hello, I'm running Zigbee2MQTT version 1.21.0 w a ConBee2 with deconz in Home Assistant, and about half of the my devices do not show an LQI (see screenshot below).  Those devices also do not show a connection in the Map tool.
I've re-paired all of the offending devices and restarted both HA and Z2M multiple times. The devices work just fine otherwise.

 I know the Deconz integration is still experimental, but is this a known issue?  

![image](https://user-images.githubusercontent.com/76019027/130373727-e923234a-47a2-4747-be9d-96f6b9f455c8.png)

**Comment by rocket59** [comment]
Are you on the latest stable conbee firmware 26680700?

**Comment by dekesone** [comment]
No it appears I am using an older firmware version 26660700.  The problem is that there doesn't seem to be a way to upgrade the firmware because of the Home Assistant operating system. The minimized environment that the HA operating system runs on doesn't include normal linux functions like systemctl so i can't run the commands necessary to upgrade.  

I previously had a Phoscan/Deconz implementation before migrating to Zigbee2MQTT and there was a tool to upgrade the firmware from within Phoscan.  Is there something similar in Z2M or is it time for me to switch away from Conbee?

**Comment by rocket59** [comment]
I do it using a Windows PC with the phoscon software. Seems easiest that way

**Reply by thecobra666** [reply]
> I do it using a Windows PC with the phoscon software. Seems easiest that way

Does upgrading the firmware require re-pairing the devices?

**Reply by dekesone** [reply]
No, updating the firmware does not require re-pairing.

Interestingly, i updated the firmware and now my LQI is showing 255 for every device.  This is after about 12 hours so each sensor has had time to report state.

**Reply by rocket59** [reply]
LQI does fall off for devices further away but everything within about 3-4m or so reports 255 for me.
